.github/workflows/maven-release.yml

```diff
@@ -36,6 +36,6 @@ jobs:
         # only for head fork
         if: github.repository == 'unicode-org/cldr'
         run: |
-          mvn -s .github/workflows/mvn-settings.xml  --file tools/pom.xml deploy
+          mvn -s .github/workflows/mvn-settings.xml  --file tools/pom.xml source:jar deploy
```

Comment:
the jar generated above wasn't still valid after the temporary version number change